_includes/navbar.html

```diff
@@ -83,9 +83,11 @@
             <!-- Blog -->
             <li{% if page.url contains '/blog/' or page.url contains '/news/' or page.url contains '/features/' %} class="active"{% endif %}><a href="{{ site.baseurl }}/blog/"><b>{{ site.data.i18n[page.language].flink_blog }}</b></a></li>
 
-            <!-- Ecosystem -->
-            <li{% if page.url contains '/ecosystem.html' %} class="active"{% endif %}><a href="{{ baseurl_i18n }}/ecosystem.html">{{ site.data.i18n[page.language].ecosystem }}</a></li>
 
+            <!-- Flink-packages -->
+            <li>
+              <a href="https://flink-packages.org" target="_blank">{{ site.data.i18n[page.language].packages }} <small><span class="glyphicon glyphicon-new-window"></span></small></a>
```

Comment:
Wondering whether `Ecosystem` isn't actually more expressive than `flink-packages.org`

Reply:
The current link name makes it pretty clear that the page is external.